aafd/untrusted_app_27.te

```diff
@@ -1 +1,16 @@
 get_prop(untrusted_app_27, vendor_video_codec_prop)
+allow untrusted_app_27 sysfs_net:dir search;
+allow untrusted_app_27 sysfs_rtc:dir search;
+allow untrusted_app_27 sysfs_rtc:file read;
+allow untrusted_app_27 sysfs_rtc:file open;
+allow untrusted_app_27 sysfs_rtc:file getattr;
+allow untrusted_app_27 vendor_file:file execute;
+allow untrusted_app_27 su_exec:file getattr;
+allow untrusted_app_27 sysfs:dir read;
+allow untrusted_app_27 vendor_file:file { map open read };
+allow untrusted_app_27 vendor_file:file getattr;
```

Comment:
Have you figured out what this vendor_file is? you can get this from the relevant avc error message.

Reply:
it was houdini

Reply:
@ceiba1985  avc error without policy for vendor_file: 12-01 14:30:20.914 18404 18404 W com.xes.jazhanghui.activity: type=1400 audit(0.0:6667): avc: denied { execute } for comm=4173796E635461736B202331 name="houdini" dev="dm-4" ino=31 scontext=u:r:untrusted_app_27:s0:c125,c256,c512,c768 tcontext=u:object_r:vendor_file:s0 tclass=file permissive=0 app=com.xes.jazhanghui.activity